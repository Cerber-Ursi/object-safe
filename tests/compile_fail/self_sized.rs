use object_safe::object_safe;

#[object_safe]
trait SelfSized where Self: Sized {}

fn main() {}
