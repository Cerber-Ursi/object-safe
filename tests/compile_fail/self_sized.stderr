error: Trait with Self: Sized trait bound cannot be made object-safe
 --> $DIR/self_sized.rs:4:23
  |
4 | trait SelfSized where Self: Sized {}
  |                       ^^^^^^^^^^^
